catalog/spatial/registry/registry-ebrim-transformer/src/main/java/org/codice/ddf/registry/converter/RegistryPackageConverter.java

```diff
@@ -199,34 +197,34 @@ private static void parseRegistryOrganization(OrganizationType organization,
 
         if (organization.isSetName()) {
             setMetacardStringAttribute(INTERNATIONAL_STRING_TYPE_HELPER.getString(organization.getName()),
-                    RegistryObjectMetacardType.ORGANIZATION_NAME,
+                    ContactAttributes.POINT_OF_CONTACT_NAME,
```

Comment:
Can use just `Contact.POINT_OF_CONTACT_NAME` here, as well as in similar cases below this.